content/docs/1.5.4/high-availability/auto-remount-volume.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Auto Remount Volume By Deleting Workload Pod
+weight: 1
+---
+
+Under following conditions, Longhorn automatically apptempts to deletes the workload pod if the pod is managed by a controller (e.g. deployment, statefulset, daemonset, etc...). By deleting the pod, its controller restarts the pod and Kubernetes handles volume reattachment and remount.
+
+1. Unexpected detachment happens, which can happen during a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703), a [Docker reboot](https://github.com/longhorn/longhorn/issues/686), a network disconnection, or a volume engine crashing.
+2. Automatic Salvage. Volumes will be automatically salvaged when all the replicas become faulty e.g. due to network disconnection. Longhorn will try to figure out which replicas are usable, then use them for the volume.
+3. Share manager goes into error state when using RWX volume.
+
+If you don't want Longhorn to automatically delete the workload pod, you can set it in [the setting `Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly`](../../references/settings#automatically-delete-workload-pod-when-the-volume-is-detached-unexpectedly) in Longhorn UI.
+
+For the pods that don't have a controller, Longhorn doesn't delete them because if Longhorn does, no one will restart them.
+To recover unexpectedly detached volumes, you would have to manually delete and recreate the pods that don't have a controller.
```

Comment:
```suggestion
If you want to prevent Longhorn from automatically deleting workload pods, disable the setting [*Automatically Delete Workload Pod when The Volume Is Detached Unexpectedly*](../../references/settings#automatically-delete-workload-pod-when-the-volume-is-detached-unexpectedly) on the Longhorn UI.

Longhorn does not delete pods without a controller because such pods cannot be restarted after deletion. To recover volumes that are unexpectedly detached, you must manually delete and restart the pods without a controller.
```